acs-integration-tests/src/test/java/com/ge/predix/acceptance/test/ACSAcceptanceIT.java

```diff
@@ -79,74 +72,51 @@
-    private boolean registerWithZac;
+    private HttpHeaders headersWithZoneSubdomain = new HttpHeaders();
+    private NewUaaTestUtil uaaTestUtil;
+    private String acsZoneName;
 
     @BeforeClass
-    public void setup() throws JsonParseException, JsonMappingException, IOException {
-        if (Arrays.asList(this.env.getActiveProfiles()).contains("public")) {
-            setupPublicACS();
-        } else {
-            setupPredixACS();
-        }
-        this.headersWithZoneSubdomain = new HttpHeaders();
-        this.headersWithZoneSubdomain.set("Predix-Zone-Id", this.testZoneSubdomain);
-    }
+    public void setup() throws IOException {
+        this.acsZoneName = this.zoneHelper.getRandomZoneName(this.getClass().getSimpleName());
+        this.headersWithZoneSubdomain.set("Predix-Zone-Id", acsZoneName);
+        this.uaaTestUtil = new NewUaaTestUtil(this.acsRestTemplateFactory, this.uaaUrl);
 
-    private void setupPredixACS() throws JsonParseException, JsonMappingException, IOException {
-        this.zacTestUtil.assumeZacServerAvailable();
+        this.acsAdminRestTemplate = this.uaaTestUtil
+                .createAcsAdminClientAndGetTemplate(Collections.singletonList(this.acsZoneName));
 
-        this.acsAdminRestTemplate = this.acsRestTemplateFactory.getACSTemplateWithPolicyScope();
-        this.registerWithZac = true;
-        this.testZoneSubdomain = this.zoneHelper
-                .createTestZone(this.acsAdminRestTemplate, this.acsZone1Name, this.registerWithZac).getSubdomain();
-    }
+        this.zoneHelper.createTestZone(this.acsAdminRestTemplate, this.acsZoneName, false);
 
-    private void setupPublicACS() throws JsonParseException, JsonMappingException, IOException {
-        UaaTestUtil uaaTestUtil = new UaaTestUtil(this.acsRestTemplateFactory.getOAuth2RestTemplateForUaaAdmin(),
-                this.uaaUrl);
-        uaaTestUtil.setup(Arrays.asList(new String[] { this.acsZone1Name, this.acsZone2Name, this.acsZone3Name }));
+        if (this.zacHelperUtil != null) {
+            zacHelperUtil.setup(this.acsZoneName, Collections.singletonMap("trustedIssuerIds",
+                    Collections.singletonList(this.uaaUrl + "/oauth/token")));
+        }
 
-        this.acsAdminRestTemplate = this.acsRestTemplateFactory.getOAuth2RestTemplateForAcsAdmin();
-        this.registerWithZac = false;
-        this.testZoneSubdomain = this.zoneHelper
-                .createTestZone(this.acsAdminRestTemplate, this.acsZone1Name, this.registerWithZac).getSubdomain();
     }
 
-    @Test(groups = { "acsHealthCheck" })
+    @Test
     public void testAcsHealth() {
 
         RestTemplate restTemplate = new RestTemplate();
         try {
-            ResponseEntity<String> heartbeatResponse = restTemplate.exchange(this.acsBaseUrl + "/monitoring/heartbeat",
-                    HttpMethod.GET, new HttpEntity<>(this.headersWithZoneSubdomain), String.class);
+            ResponseEntity<String> heartbeatResponse = restTemplate
```

Comment:
I am assuming all these are formatting changes only?  Please verify that your local IDE rules are indeed the right ones.